lexos/managers/file_manager.py

```diff
@@ -1236,24 +1236,29 @@ def get_matrix_deprec(self, use_word_tokens: bool, use_tfidf: bool,
     def get_class_division_map(self) -> pd.DataFrame:
         """:return: a panda frame that contains class division map."""
 
-        # active files labels and classes
+        # active files labels and classes.
         active_files = self.get_active_files()
         file_labels = [file.label for file in active_files]
-        class_labels = {file.class_label for file in active_files}
+        class_labels = list({file.class_label for file in active_files})
```

Comment:
why do this have to be a list? if it is a list why not just use list comprehension?